Mathlib/LinearAlgebra/Matrix/SesquilinearForm.lean

```diff
@@ -683,21 +689,22 @@ variable [DecidableEq ι] [Fintype ι]
 
 theorem _root_.Matrix.separatingLeft_toLinearMap₂'_iff_separatingLeft_toLinearMap₂
     {M : Matrix ι ι R₁} (b : Basis ι R₁ M₁) :
-    M.toLinearMap₂'.SeparatingLeft (R := R₁) ↔ (Matrix.toLinearMap₂ b b M).SeparatingLeft :=
+    (Matrix.toLinearMap₂' (R := R₁) M).SeparatingLeft (R := R₁) ↔
```

Comment:
It no longer appears possible to use the dot notation on `M` here. Is this fixable @eric-wieser ?